src/OpenTelemetry.Exporter.OpenTelemetryProtocol/OtlpExporterOptions.cs

```diff
@@ -44,16 +44,19 @@ public class OtlpExporterOptions
 
         internal readonly Func<HttpClient> DefaultHttpClientFactory;
 
+        private const string DefaultGrpcEndpoint = "http://localhost:4317";
+        private const string DefaultHttpProtobufEndpoint = "http://localhost:4318";
```

Comment:
minor: this could be renamed to `DefaultHttpEndpoint` since this default would be the same once we support the OTLP/HTTP json payload format.

Reply:
fixed.